### ISSUE TITLE
autoleveling and manual tool change tutorial german

### ISSUE THREAD

**Issue by ltspicer** [vlachoudis/bCNC#1614]
I wrote a german tutorial for autoleveling and manual tool change.
If you want, you can transfer it to the wiki.
I hope I have made a useful contribution to bCNC.
[bcnc_autoleveling_anleitung.pdf](https://github.com/vlachoudis/bCNC/files/7123569/bcnc_autoleveling_anleitung.pdf)

**Comment by Harvie** [comment]
@treki1 thank you, you should have write access to the wiki, so you can add it yourself if you wish...

**Comment by treki1** [comment]
How can I upload pictures to include in the wiki?

**Comment by treki1** [comment]
Uploads are disabled :(

![Auswahl_001](https://user-images.githubusercontent.com/11616336/132541280-eefe08a4-c28c-47bd-b1c4-9ce2202939f7.png)

**Comment by Harvie** [comment]
Yes, you have to do the same thing as with the translation. Unfortunately :-(
I don't know why github does not allow to upload images directly to the wiki...

https://github.com/treki1/bCNC/tree/doc

**Comment by treki1** [comment]
Done ^^
https://github.com/vlachoudis/bCNC/wiki/Autoleveling-und-manueller-Werkzeugwechsel-mit-bCNC

**Comment by Harvie** [comment]
please also make pull request from https://github.com/treki1/bCNC/tree/doc to https://github.com/vlachoudis/bCNC/tree/doc
Otherwise the files will disappear when you update your fork to reflect the upstream.

**Comment by ltspicer** [comment]
Is this ok?
https://github.com/vlachoudis/bCNC/pull/1617

**Comment by ltspicer** [comment]
Why is the pdf version deleted?
https://github.com/vlachoudis/bCNC/wiki/Autoleveling-und-manueller-Werkzeugwechsel-mit-bCNC
At the bottom of the document.

**Comment by ImranKhan203** [comment]
Hello,

Please help me out.
I am using GRBL 0.9 with the help of bCNC software and I want to attach Tower Lamp/Stack Light on our CNC machine.
I would like to add 2 outputs on our machine to indicate if we’re running the program (green light) or if we’re in pause (called ‘hold’ in the program / red light). For that, I have to work on bCNC that is the software used in all our machines.

So, when the program is running, the green light has to be lighted, and if we’re in pause, the red light has to be lighted. The orange one will probably be used for something else in the future.
To do that, I’ll have to detect the state of the program and send it to the lights (2 outputs).

Please tell me in which file of bCNC and which type of program I have to use?

Here is the file which I am using on Python 2.7.16 :
[bCNC01.ZIP](https://github.com/vlachoudis/bCNC/files/7245704/bCNC01.ZIP)


I want to attach this type of Tower Lamp/Stack Light :
![tower-lamp-230v-ac-500x500](https://user-images.githubusercontent.com/91553705/135122090-b0f3c0db-ffb0-486a-b9ad-71684dd09ecb.jpg)


Please suggest me in which file of bCNC and which type of program I have to use?

**Comment by ltspicer** [comment]
Pls create a new (issue) post.
